### ISSUE TITLE
message.channel.sendMenu(changeColour)  is not a function

### ISSUE THREAD

**Issue by GalCohenDev** [Mundayne/reaction-core#11]
```js
client.on('message', message => {
  if (!message.author.bot && message.content === 'rc!test') {
    message.channel.sendMenu(changeColour)
  }
})
```

**sendMenu()** is not a function what else do I use?

**Comment by Mundayne** [comment]
I'm unable to replicate this error; what version of discord.js and Node.JS are you using?
If this is the default `example.js` file, make sure that the line `const RC = require('./index')` is present.

**Comment by GalCohenDev** [comment]
Node v12, discord.js v12.2, do I need to add this line, `const RC = require('./index')` to my project?
(I didnt download example.js, I followed your guide)

**Comment by Mundayne** [comment]
Aha, after a deeper investigation I've found the issue - somewhere along the line something has changed about adding methods to prototypes in NodeJS modules, so the `addMenu` method is never being added to the channels. I'll work on fixing this, but it may take a while so in the meantime you can use this code to send menus:

```js
message.channel.send(typeof changeColour.text === 'string' ? changeColour.text : { embed: changeColour.text }).then(async m => {
    for (let button in changeColour.buttons) {
        await m.react(button).catch(console.error)
    }
    changeColour.register(m)
})
```

You'll just have to change `changeColour` to the menu object you want to send, I did the above example for the tutorial case.

**Comment by GalCohenDev** [comment]
Thx bro works like a charm!

**Comment by Mundayne** [comment]
This has been fixed in 2.4.0.
